.circleci/config.yml

```diff
@@ -445,6 +463,8 @@ jobs:
       - MAGMA_MODULES_FILE=/home/circleci/project/circleci/modules.yml
     steps:
       - checkout
+      - build/determinator:
+          <<: *orc8r_build_verify
```

Comment:
this should be all modules because orc8r build pulls in go code from all of them